coalib/processes/BearRunning.py

```diff
@@ -1,37 +1,39 @@
+import logging
 import queue
 import traceback
 
 from coalib.bears.BEAR_KIND import BEAR_KIND
 from coalib.bears.GlobalBear import GlobalBear
 from coalib.bears.LocalBear import LocalBear
 from coalib.misc import Constants
-from coalib.processes.communication.LogMessage import LOG_LEVEL, LogMessage
+from coalib.processes.communication.LogMessage import LOG_LEVEL
 from coalib.processes.CONTROL_ELEMENT import CONTROL_ELEMENT
 from coalib.results.Result import Result
 
 
-def send_msg(message_queue, timeout, log_level, *args, delimiter=' ', end=''):
+def send_msg(log_level, *args, delimiter=' ', end='', message_queue=None,
+             timeout=None):
     """
     Puts message into message queue for a LogPrinter to present to the user.
 
-    :param message_queue: The queue to put the message into and which the
-                          LogPrinter reads.
-    :param timeout:       The queue blocks at most timeout seconds for a free
-                          slot to execute the put operation on. After the
-                          timeout it returns queue Full exception.
     :param log_level:     The log_level i.e Error,Debug or Warning.It is sent
                           to the LogPrinter depending on the message.
     :param args:          This includes the elements of the message.
     :param delimiter:     It is the value placed between each arg. By default
                           it is a ' '.
     :param end:           It is the value placed at the end of the message.
+    :param message_queue: The queue to put the message into and which the
+                          LogPrinter reads.
+    :param timeout:       The queue blocks at most timeout seconds for a free
+                          slot to execute the put operation on. After the
+                          timeout it returns queue Full exception.
     """
     output = str(delimiter).join(str(arg) for arg in args) + str(end)
-    message_queue.put(LogMessage(log_level, output),
-                      timeout=timeout)
+    logging.log(log_level, output)
 
 
-def validate_results(message_queue, timeout, result_list, name, args, kwargs):
+def validate_results(result_list, name, args, kwargs, message_queue=None,
```

Comment:
as above.  deprecate does not mean break API

Reply:
@jayvdb but `validate_results` is only used in BearRunning.py and it makes only one call in `run_bear` method, how is that breaking the API?